apps/extension/src/popup/index.ts

```diff
@@ -0,0 +1,3 @@
+import App from "../App";
```

Comment:
We may not use a popup, but instead inject our app into the page. This is here mainly to confirm that the build-pipeline is functioning as it should.